src/modules/epsr/process.cpp

```diff
@@ -424,7 +424,7 @@ Module::ExecutionResult EPSRModule::process(ModuleContext &moduleContext)
 
             // Always add absolute data to the scattering matrix - if the calculated data has been normalised, remove this
             // normalisation from the reference data (we assume that the two are consistent)
-            auto normType = module->keywords().getEnumeration<StructureFactors::NormalisationType>("NormaliseTo");
+            auto normType = module->keywords().getEnumeration<StructureFactors::NormalisationType>("normaliseTo");
```

Comment:
Bit of aggressive search/replacing!
```suggestion
            auto normType = module->keywords().getEnumeration<StructureFactors::NormalisationType>("NormaliseTo");
```

Reply:
Agh!...